test/sampleClassGtest.cpp

```diff
@@ -13,5 +13,5 @@ using namespace testing;
 
```

Comment:
You can delete this file if you want (or keep it like this... it doesn't really matter :) )